src/pages/programs/program-solution-entity/program-solution-entity.ts

```diff
@@ -79,17 +79,11 @@ export class ProgramSolutionEntityPage {
 
```

Comment:
This page is changed every where for submissionId and submissionStatus to _id and status